lib/logstash/plugin_mixins/common.rb

```diff
@@ -0,0 +1,93 @@
+module LogStash
+  module PluginMixins
+    module KafkaAvroSchemaRegistry
+      DEFAULT_SERIALIZER_CLASS = "org.apache.kafka.common.serialization.StringDeserializer"
```

Comment:
nit: I would move this constant to `LogStash::Inputs::Kafka`, since it has more to do with the output than the schema registry